### ISSUE TITLE
Option for view only session

### ISSUE THREAD

**Issue by bandicootdancing** [screego/server#127]
A request to allow hosts to choose a view only session. The host specifically can share their screen while the share button is disabled for other users. This is to help prevent people from accidentally sharing their screen.

Another option would be a setting where users can request to share their screen during a session when it's actually needed.

**Comment by jolinux** [comment]
Yes view-mode a nice feature to "broadcast" only my desktop or window !
